components/Padrinho/PadrinhoContent.js

```diff
@@ -0,0 +1,28 @@
+import Link from 'next/link'
```

Comment:
Rename the file to Pedrinho.js (each component needs a root file with the same folder name (index).

Reply:
Done